src/backend/marsha/core/services/video_participants.py

```diff
@@ -0,0 +1,23 @@
+"""Services for video live participants."""
+
+
+def add_participant_asking_to_join(video, participant):
+    """Add a participant asking to join a video."""
+    if participant in (
+        video.participants_asking_to_join or video.participants_in_discussion
+    ):
+        return False
```

Comment:
I would not return a boolean but raise an exception if the participant is already present.

Reply:
Ok, should we respond with a 400 in that case ?

Reply:
I would say yes